src/db/schemas/Timesheet.ts

```diff
@@ -40,15 +40,16 @@ export const TimeEntrySchema = z.object({
     @PTO - Cell signifying paid time off (PTO) 
 */
 export enum CellType {
-  REGULAR = "Regular", 
+  REGULAR_LEGACY = "Regular", // No longer using this format for data, but some older timesheet entries may have the 'legacy' type
```

Comment:
I think it's ok to just delete any old sheets since its all test data right now

Reply:
I think we should actually keep this in for now, just because it's been so long since this PR was initially opened that there's more test data now...